provider/pkg/gen/utilities_test.go

```diff
@@ -40,7 +40,7 @@ var apiPropToSdkNames = map[string]string{
 
 func TestApiPropNameToSdkName(t *testing.T) {
 	for name, expected := range apiPropToSdkNames {
-		actual := apiPropNameToSdkName(name)
+		actual := apiPropNameToSdkName("typeName", name)
```

Comment:
Nit - maybe add the `Project` type name to the tests while you are at it?

Reply:
Have just added the exception case to the test - all other types will be treated the same.